classes/controller/FrontController.php

```diff
@@ -1465,7 +1465,7 @@ public function getTemplateVarUrls()
             }
 
             foreach ($assign_array as $assign_key => $assign_value) {
-                if (substr($assign_value, 0, 1) == '/' || $this->ssl) {
+                if ($this->ssl) {
```

Comment:
Was it reported by PHPStan? What was the reason?

Reply:
Yes, it was reported by PHPStan.

The reason was that `substr($assign_value, 0, 1) == '/'` was always `true`. It's normal because `assign_array` are a string array with path so PHPStan is right.

Reply:
Well I guess you should do this then:
```diff
            foreach ($assign_array as $assign_key => $assign_value) {
-                if (substr($assign_value, 0, 1) == '/' || $this->ssl) {
-                    $urls[$assign_key] = $http . Tools::getMediaServer($assign_value) . $assign_value;
+               $urls[$assign_key] = $http . Tools::getMediaServer($assign_value) . $assign_value;
-                } else {
-                    $urls[$assign_key] = $assign_value;
-                }
            }
```

Reply:
Fixed. Thanks @atomiix